.github/workflows/ansible-test.yml

```diff
@@ -117,10 +117,10 @@ jobs:
           python-version: ${{ matrix.python_version }}
 
       - name: Install community.general
-        run: ansible-galaxy collection install community.general -p ../../
+        run: "for i in {1..3}; do ansible-galaxy collection install community.general -p ../../ && break || sleep 5; done"
```

Comment:
I think we should get rid of the retry loops here. Better solved at galaxy end and caching.

Reply:
Oops. I didn't mean to add these here. removing...